### PR TITLE
update: adds docs and xlsx tests for DEGREES and RADIANS functions

### PR DESCRIPTION
**Pull request by elsaminsut**
This PR adds tests for the DEGREES and RADIANS functions. These include xlsx tests with edge cases. It also includes new documentation pages for these functions.

See issue #494

### CODE REVIEW

File: docs/src/functions/math_and_trigonometry/degrees.md [hunk 1]
```diff
@@ -4,9 +4,38 @@ outline: deep
...
+DEGREES returns a [number](/features/value-types#numbers) that represents the value of the given angle expressed in degrees.
+
+### Error conditions
+* In common with many other IronCalc functions, DEGREES propagates errors that are found in its argument.
+* If no argument, or more than one argument, is supplied, then DEGREES returns the [`#ERROR!`](/features/error-types.md#error) error.
+* If the value of the *angle* argument is not (or cannot be converted to) a [number](/features/value-types#numbers), then DEGREES returns the [`#VALUE!`](/features/error-types.md#value) error.
+<!--@include: ../markdown-snippets/error-type-details.txt-->
+
+## Examples
+[See some examples in IronCalc](https://app.ironcalc.com/?example=degrees).
```
Comment (nhatcher) [inline docs/src/functions/math_and_trigonometry/degrees.md:36]: You need to add that example right?

Comment (elsaminsut) [reply docs/src/functions/math_and_trigonometry/degrees.md:36]: I removed the links instead until I start working on the example files

**Review by nhatcher — APPROVED** [review]
LGTM!
